openslides/agenda/migrations/0003_auto_20170818_1202.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.10.7 on 2017-08-18 10:02
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
```

Comment:
I think this won't work for existing databases because the new permission won't be generated.

Reply:
Maybe I am wrong here. I have to check it.

Reply:
@normanjaeckel Successfully tested the migration from master to this branch. After migrate the new permission was visible (and unchecked) in the permission matrix.